write.go

```diff
@@ -112,10 +113,10 @@ func write(w *bufio.Writer, a interface{}) {
 // allocations that occur when building the data structures representing the
 // kafka protocol requests.
 
-func writeFetchRequestV1(w *bufio.Writer, correlationID int32, clientID, topic string, partition int32, offset int64, minBytes, maxBytes int, maxWait time.Duration) error {
+func writeFetchRequestV2(w *bufio.Writer, correlationID int32, clientID, topic string, partition int32, offset int64, minBytes, maxBytes int, maxWait time.Duration) error {
```

Comment:
Do you know which version of Kafka we're dropping backward compatibility with by moving to FetchRequest v2?